modules/modelSetup/BaseStableDiffusionXLSetup.py

```diff
@@ -223,6 +304,8 @@ def calculate_loss(
     ) -> Tensor:
         predicted = data['predicted']
         target = data['target']
+        predicted = predicted[:, :3, :, :]
+        target = target[:, :3, :, :]
```

Comment:
SD uses a 4 dimensional latent space. The 4 channels are completely independent from RGB values. with these two lines you cut off one of the channels, meaning you will ignore the loss of one of the 4 channels. This might lead to reduced learning at best, and might completely fry the model if you are unlucky.